src/test/resources/fake-agency/transfers.txt

```diff
@@ -1 +1,2 @@
-from_stop_id,to_stop_id,transfer_type,min_transfer_time
+from_stop_id,to_stop_id,from_trip_id,to_trip_id,from_route_id,to_route_id,transfer_type,min_transfer_time
```

Comment:
Should there be a case for when not all columns are populated? (e.g. for a more generic transfer policy)

Reply:
Yes, I have added.